src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -805,7 +816,16 @@ MulticopterAttitudeControl::run()
 
 			bool attitude_setpoint_generated = false;
 
-			if (_v_control_mode.flag_control_attitude_enabled && _vehicle_status.is_rotary_wing) {
+			// vehicle is hovering
+			bool is_hover = _vehicle_status.is_rotary_wing && !_vehicle_status.in_transition_mode;
```

Comment:
That's better  but instead of adding the comment "vehicle is hovering", just name the variable `is_hovering` and it's clear.

Also, these `bool`s can be `const`.